src/main/java/uk/gov/hmcts/cmc/claimstore/controllers/DocumentsController.java

```diff
@@ -73,9 +74,10 @@ public DocumentsController(DocumentsService documentsService) {
     public ResponseEntity<ByteArrayResource> countyCourtJudgement(
         @ApiParam("Claim external id")
         @PathVariable("externalId") @NotBlank String externalId,
-        @RequestHeader(HttpHeaders.AUTHORIZATION) String authorisation
+        @RequestHeader(HttpHeaders.AUTHORIZATION) String authorisation,
+        @RequestParam(name = "issue", required = false) boolean issue
     ) {
-        byte[] pdfDocument = documentsService.generateCountyCourtJudgement(externalId, authorisation);
+        byte[] pdfDocument = documentsService.generateCountyCourtJudgement(externalId, authorisation, issue);
```

Comment:
Stop passing this boolean flag around. We aim to look into case details to check whether CCJ was issued or just requested. Check with @dharmendrak for details.